kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/data/DefaultRowBasedColumnarBatch.java

```diff
@@ -227,6 +227,11 @@ public <T> List<T> getArray(int rowId) {
             return rows.get(rowId).getArray(columnOrdinal);
         }
 
+        @Override
+        public ColumnVector getChild(int ordinal) {
+            return ColumnVector.super.getChild(ordinal);
```

Comment:
hm, how does this work? if `DefaultRowBasedColumnarBatch` doesn't even extend ColumnVector?

i think my java is rusty

Reply:
like how does the super of `ColumnVector` know anything about the internal data of this instance? in order to get the child at that ordinal?

Reply:
this is the commit that I missed to push to the PR branch: 9fb71285647c06d8fa2861445d7e66e690fe1f63